e2e/config.js

```diff
@@ -48,6 +48,13 @@ module.exports = {
     email: 'hmcts.civil+organisation.1.superuser@gmail.com',
     type: 'systemupdate'
   },
+  judgeUser: {
+    password: process.env.ENVIRONMENT === 'demo' || process.env.ENVIRONMENT ===  'preview'
+      ? defaultPassword : 'Hmcts1234',
+    email: process.env.ENVIRONMENT === 'demo' || process.env.ENVIRONMENT ===  'preview'
```

Comment:
The condition says if env is preview or demo then user judge-civil-02@example.com.. i think the condiiton should be if env is local use judge-civil-02@example.com and all other envs use 4924246EMP-@ejudiciary.net